### PR TITLE
Migrate Detect Language

### PR DESCRIPTION
**Pull request by abpai94**
#19

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/11.0/phpunit.xsd">
+<phpunit
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  colors="true"
+  xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/11.0/phpunit.xsd"
+  displayDetailsOnTestsThatTriggerWarnings="true"
```
Comment (davidcoutadeur) [inline phpunit.xml:6]: Even if this is not a big modification, I am not fond of doing this in this issue. Could you remove these modifications from your commit, or explain why you need them?

File: tests/Ltb/LanguageTest.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+require __DIR__ . '/../../vendor/autoload.php';
+
+use PHPUnit\Framework\TestCase;
+
+final class LanguageTest extends \Mockery\Adapter\Phpunit\MockeryTestCase
+{
+    function test_accept_all_language() {
+        # User-Agent Language
+        $_SERVER['HTTP_ACCEPT_LANGUAGE'] = "ar;q=1, cn;q=1, de;q=1, el;q=1, es;q=1, fr;q=1, it;q=1, nb-NO;q=1, pl;q=1, pt-PT;q=1, ru;q=1, sl;q=1, tr;q=1, zh-CN;q=1, ca;q=1, cs;q=1, ee;q=1, en;q=1, eu;q=1, hu;q=1, ja;q=1, nl;q=1, pt-BR;q=1, rs;q=1, sk;q=1, sv;q=1, uk;q=1, zh-TW;q=1";
+
```
Comment (davidcoutadeur) [inline tests/Ltb/LanguageTest.php:11]: please remove all extra spaces at the end of lines

**Review by davidcoutadeur — CHANGES_REQUESTED** [review]
Globally, it seems good to me! Thanks for the unit tests.

Some minor changes requested. (see the review)

Also: please squash your commits into one

File: phpunit.xml (hunk 1 above)
Comment (abpai94) [reply phpunit.xml:6]: When running tests with warnings or deprecations there are no details to explain where these warnings are being triggered. These flags allow the cause of the warnings and deprecations to be displayed in more detail. It helps facilitate the debugging of tests more easily.

File: tests/Ltb/LanguageTest.php (hunk 2 above)
Comment (abpai94) [reply tests/Ltb/LanguageTest.php:11]: Removed the long list of languages as it is an unrealistic representation of a User-Agent Accept-Language variable.

Comment (davidcoutadeur) [reply tests/Ltb/LanguageTest.php:11]: Ok for your last change, but please also remove extra spaces at the end of lines.

**Comment by abpai94** [comment]
@davidcoutadeur this is ready to be merged after implementing tests and your review comments.

**Comment by abpai94** [comment]
@davidcoutadeur implemented changes, ready to be merged!

**Pull request merged**